src/nodevisitor.rs

```diff
@@ -48,6 +59,19 @@ impl<'a> StaticPathParser<'a> {
         visitor.visit(&node);
         Ok(visitor)
     }
+
+    pub fn get_path_info(path: &'a str) -> PathInfoFlags {
+        Self::check(path).map_or_else(
+            |_| PathInfoFlags::INVALID,
+            |parser| {
+                if parser.valid == VisitStatus::Valid {
+                    PathInfoFlags::STATIC | PathInfoFlags::DEFINED_ORDER
```

Comment:
@MeirShpilraien We currently do not differentiate between `STATIC` and `DEFINED_ORDERED`.
We currently do not accept, in `StaticPathParser`, Unions and closed positive Slices (Ranges),
Which could also be considered `STATIC` but might not be considered `DEFINED_ORDERED` if they appear more than once,
e.g., path with `[a,b][c,d]` could be traversed as `[a,c], [a,d], [b,c], [b,d]` or as `[a,c], [b,c], [a,d], [b,d]`,
So order is not well defined, although it is static.